src/compiler/crystal/types.cr

```diff
@@ -311,16 +311,17 @@ module Crystal
     # the types are in the same namespace. Otherwise, it means they are just
     # in the same type hierarchy.
     def has_protected_acces_to?(type, allow_same_namespace = true)
```

Comment:
```suggestion
    def has_protected_access_to?(type, allow_same_namespace = true)
```
found a  typo :eyes:

Reply:
Oh, yes! I saw it too but while fixing the bug I completely forgot about it. I'll add another commit. Thanks!